.github/actions/build-backend/action.yml

```diff
@@ -15,6 +15,9 @@ inputs:
     default: false
   sp-creds:
     description: "Azure Service Principal creds"
+  skip-schema-validation:
+    description: "Skip schema validation step"
+    default: false
```

Comment:
Based on the name "skip-schema-validation" it would make more sense to invert the logic, meaning default: should be TRUE, and then update the logic on line 86 to be `!inputs.skip-schema-validation`